wrapper/pyproject.toml

```diff
@@ -12,7 +12,7 @@ authors = [
 ]
 description = "Type hints for Abaqus/Python scripting"
 readme = "README.md"
-dynamic = ["version"]
+dynamic = ["version", "dependencies"]
```

Comment:
https://packaging.python.org/en/latest/specifications/pyproject-toml/#declaring-project-metadata-the-project-table